opm/autodiff/ISTLSolverEbos.hpp

```diff
@@ -51,11 +53,112 @@ NEW_PROP_TAG(Scalar);
 NEW_PROP_TAG(GlobalEqVector);
 NEW_PROP_TAG(SparseMatrixAdapter);
 NEW_PROP_TAG(Indices);
+NEW_PROP_TAG(Simulator);
+NEW_PROP_TAG(EclWellModel);
 
 END_PROPERTIES
 
 namespace Opm
 {
+//=====================================================================
+// Implementation for ISTL-matrix based operator
+//=====================================================================
+
+/*!
+   \brief Adapter to turn a matrix into a linear operator.
+
+   Adapts a matrix to the assembled linear operator interface
+ */
+  template<class M, class X, class Y, class WellModel, bool overlapping,class TypeTag >
+class WellModelMatrixAdapter : public Dune::AssembledLinearOperator<M,X,Y>
```

Comment:
Seems like the code was not removed as guessed above, but moved. Assuming this for all places with "same here", too,